pkg/plugins/manager/loader/assetpath/assetpath.go

```diff
@@ -153,3 +153,18 @@ func getBaseDir(pluginDir string) string {
 	}
 	return baseDir
 }
+
+func (s *Service) GetLocales(n PluginInfo) (map[string]string, error) {
+	pathToLocales, err := s.RelativeURL(n, "locales/")
+	if err != nil {
+		return nil, fmt.Errorf("get locales: %w", err)
+	}
+
+	// loop through all the locales specified in the plugin.json and add them to the list
+	locales := map[string]string{}
+	for _, locale := range n.pluginJSON.Locales {
+		locales[locale] = fmt.Sprintf("%s/%s/%s.json", pathToLocales, locale, n.pluginJSON.ID)
```

Comment:
nit: can you use `path.Join` instead?

Reply:
Thanks for suggesting this and I started out with this but I couldn't use it in the end because `path.Join` cleans the path as a last moment and then any CDN urls would become corrupted.

Reply:
You should be able to just use `url.JoinPath` like in the test I believe?

```go
locales[locale], err = url.JoinPath(pathToLocales, locale, fmt.Sprintf("%s.json", n.pluginJSON.ID))
if err != nil {
    return nil, fmt.Errorf("join path: %w", err)
}
```

Reply:
Aaah, great suggestion! I'll have a look 👍

Reply:
Fixed 👍